test/bats/plugin-test.bats

```diff
@@ -206,6 +206,69 @@ SLEEP_TIME=1
     assert_success
 }
 
+@test "namespaced sbom/notary/cosign/licensechecker/schemavalidator verifiers test" {
```

Comment:
just a note that the other all-in-one test is very flaky. The concurrent oras store issue has a higher chance.

Reply:
thanks for the note! It did fail quite often recently, I added namespaced test to all-in-one setup so that we can cover most namespaced resources to increase our confidence on multi-tenancy feature.

Reply:
I'm wondering if you think it's necessary to run an all-in-one test for this scenario. I'm concerned adding another all-in-one test will increase chances of test failures. Is there a part of all-in-one test unique to multi-tenancy that cannot be covered by a single verifier test?